README.md

```diff
@@ -2,18 +2,13 @@
 
 pyPolarionCli is a command-line tool designed for easy access to Polarion work items, e.g. for metric creation.
 
-[![License](https://img.shields.io/badge/license-bsd-3.svg)](https://choosealicense.com/licenses/bsd-3-clause/) [![Repo Status](https://www.repostatus.org/badges/latest/wip.svg)](https://www.repostatus.org/#wip) [![CI](https://github.com/NewTec-GmbH/pyPolarionCli/actions/workflows/ci.yml/badge.svg)](https://github.com/NewTec-GmbH/pyPolarionCli/actions/workflows/ci.yml)
-
-* [Installation](#installation)
-* [Overview](#overview)
-* [Usage](#usage)
-* [Commands](#commands)
-  * [Search](#search)
-* [Examples](#examples)
-* [Used Libraries](#used-libraries)
-* [Issues, Ideas And Bugs](#issues-ideas-and-bugs)
-* [License](#license)
-* [Contribution](#contribution)
```

Comment:
Why TOC removed?

Reply:
TOC is integrated and generated automatically by GitHub (top right corner)

Reply:
![image](https://github.com/user-attachments/assets/8fe96066-2c5b-433b-947f-40915e329db6)